generators/generator-base.js

```diff
@@ -2096,7 +2096,6 @@ module.exports = class extends PrivateBase {
                 `"${keytoolPath}keytool" -genkey -noprompt `
                 + '-storetype PKCS12 '
                 + '-keyalg RSA '
-                + '-alias selfsigned '
                 + `-keystore ${keyStoreFile} `
                 + '-storepass password '
```

Comment:
I'm a bit sad that the keystore password is `password` and not `changeit` as traditionally used 😄.